src/components/common/calendar/Calendar.js

```diff
@@ -0,0 +1,172 @@
+import React, { useState, useEffect } from 'react';
+// import './styles/App.css';
+import 'antd/dist/antd.css';
+import { Calendar, Modal, Badge, Button } from 'antd';
+import CalendarModal from './CalendarModal';
+import axiosWithAuth from '../../../utils/axiosWithAuth';
+
+const initialValues = [
+  {
+    date: '15/03/2022',
+    type: 'warning',
+    content: 'This is warning event.',
+    details: 'Test information 1',
+  },
+  {
+    date: '15/03/2022',
+    type: 'success',
+    content: 'This is usual event.',
+    details: 'Test information 2',
+  },
+  {
+    date: '16/03/2022',
+    type: 'error',
+    content: 'This is error event 1.',
+    details: 'Test information 3',
+  },
+  {
+    date: '16/03/2022',
+    type: 'error',
+    content: 'This is error event 2.',
+    details: 'Test information 4',
+  },
+  {
+    date: '16/03/2022',
+    type: 'error',
+    content: 'This is error event 3.',
+    details: 'Test information 5',
+  },
+  {
+    date: '12/04/2022',
+    type: 'success',
+    content: 'This is usual event1.',
+    details: 'Test information 6',
+  },
+  {
+    date: '12/04/2022',
+    type: 'success',
+    content: 'This is usual event2.',
+    details: 'Test information 7',
+  },
+];
+
+function CalendarApp() {
+  const [isModalVisible, setIsModalVisible] = useState(false);
+  const [isScheduleModalVisible, setIsScheduleModalVisible] = useState(false);
+  const [event, setEvent] = useState(null);
+  const [eventsArr, setEventsArr] = useState(initialValues);
+
+  // ****** will update with endpoint once db endpoint is built ******
+  // useEffect(() => {
+  //     axiosWithAuth.get('/')
+  //         .then(events => {
+  //             setEventsArr(events)
+  //         })
+  //         .catch(err => console.error(err))
+  // }, []);
+
+  const showModal = value => {
+    setEvent(value);
+    setIsModalVisible(true);
+  };
+
+  const handleOk = () => {
+    setIsModalVisible(false);
+  };
+
+  const handleCancel = () => {
+    setIsModalVisible(false);
+  };
+
+  const showScheduleModal = () => {
+    setIsScheduleModalVisible(true);
+  };
+
+  function getListData(value, events) {
+    let listData = [];
+    let dateValue = value.format('DD/MM/YYYY'); // you can parse value in every format you want
+
+    events.map(e => {
+      if (e.date === dateValue) {
```

Comment:
Not sure how much I care about this rule, but it's worth a refactor if ESLint is barking at it. You could simply substitute the `arrow func` for a `func declaration` ex. `function name()`